check_api/src/main/java/com/google/errorprone/hubspot/HubSpotUtils.java

```diff
@@ -103,6 +104,17 @@ public static boolean isErrorHandlingEnabled(ErrorProneOptions options) {
     return isErrorHandlingEnabled(options.getFlags());
   }
 
+  public static boolean isCanonicalSuppressionEnabled(VisitorState visitorState) {
+    ErrorProneFlags flags = visitorState.errorProneOptions().getFlags();
+    if (flags == null) {
+      return false;
+    }
+
+    return flags
+        .getBoolean("hubspot:canonical-suppression")
```

Comment:
There's an `s` on the end of the value in your PR description. Did you mean to have one here?

Reply:
Ah yea I typo'd it in the description, the flag here matches the `isCanonicalSuppressionEnabled` method name so I think i'll leave it as is. Good catch though 😄